distribution/docs/src/main/resources/_contents/_quickstart/quickstart-configuring-contents.adoc

```diff
@@ -0,0 +1,15 @@
+
+=== Configuring (Quick Install)
+
+Set the configurations needed to run ${branding}
+
+.. Go to ${secure_url}/admin.
+.. Enter the default username of `admin` and the password of `admin`.
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure guest claims attributes or use defaults.
```

Comment:
Since this is the quick start guide, should we just tell the reader to keep the defaults? If we want to keep the guest claims part, should we refer to the section that explains what those are?